openpathsampling/features/shared.py

```diff
@@ -0,0 +1,313 @@
+import copy
+
+import numpy as np
+from simtk import unit as u
+
+from openpathsampling.netcdfplus import StorableObject, ObjectStore
+
+# =============================================================================
+# SIMULATION CONFIGURATION
+# =============================================================================
+
+class Configuration(StorableObject):
```

Comment:
Renaming suggestions:

* `Configuration` => `StaticContainer`
* `Momentum` => `KineticContainer`

Both of these are essentially container classes. I think we should name them as such. It also resolves my long-standing complaint that `Momentum` isn't the momenta. This last is getting very frustrating as I work with `dynamiq`, where everything is expressed in terms of momenta, not velocities.